pkg/cmd/server/api/types.go

```diff
@@ -545,6 +545,8 @@ type IdentityProvider struct {
 	UseAsLogin bool
 	// MappingMethod determines how identities from this provider are mapped to users
 	MappingMethod string
+	// MappingMethodConfig is used to configure custom MappingMethods.
```

Comment:
Can you be more specific.  Is this a map of MappingMethod to filenames?  What about a structure that would allow me to specify a filename or inline another API type?